lib/LedgerSMB/X12.pm

```diff
@@ -35,7 +35,7 @@ use LedgerSMB::Magic qw( EDI_PATHNAME_MAX );
 use LedgerSMB::Sysconfig;
 use DateTime;
 
-my $counter = 1000; #for 997 generation
+my $counter = 1000; #for 997 generation  ## no critic (ProhibitMagicNumbers) sniff
```

Comment:
This counter isn't used in this file. Maybe we don't need to put a "no critic" on it, but instead just remove it?

Reply:
I considered that.   But my thinking went:  What's '997 generation'?  Aspiring or dying code?  Don't know.

I dislike no-criticing, but not finding the time to fix makes it compelling to avoid more
infractions.

Reply:
We missed the use of `${counter}`